mappings/net/minecraft/nbt/AbstractNbtList.mapping

```diff
@@ -0,0 +1,20 @@
+CLASS net/minecraft/class_2483 net/minecraft/nbt/AbstractNbtList
+	COMMENT Represents an abstraction of a mutable NBT list which holds elements of the same type.
+	METHOD add (ILjava/lang/Object;)V
+		ARG 1 index
+		ARG 2 value
+	METHOD method_10533 addElement (ILnet/minecraft/class_2520;)Z
+		ARG 1 index
+		ARG 2 element
+	METHOD method_10535 setElement (ILnet/minecraft/class_2520;)Z
```

Comment:
```suggestion
	METHOD method_10533 add (ILnet/minecraft/class_2520;)Z
		ARG 1 index
		ARG 2 element
	METHOD method_10535 set (ILnet/minecraft/class_2520;)Z
```

The synthetic `void add(int, Object)` and `set(int, Object)` generated by javac to override the superclass methods call these, so per javac behavior they are named so

Reply:
those are not similar to the superclass ones as it returns boolean value if the passed element has been successfully added (if it's the good type)

superclass set is `<T> T set(int index, T value)` and `method_10535` is `boolean set(int index, NbtElement element)` but I can agree with renaming those still but I feel like we lose some explicit naming.